### DISCUSSION TITLE
Weird spectrogram (solved)

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#698] [Q&A] [answered]
Hi folks !

I've update Birdnet-Pi this morning (my current problem may or may not be linked to that specific point, I don't know...) and my spectrogram now looks like a very thin line

![image](https://user-images.githubusercontent.com/112718877/212550861-19defc0e-27a6-4ed5-aea0-097941013c91.png)

It seems like there's been no detection since then.

I've tried changing recording/extraction lengths but it was helpless.

BTW, live stream is still working fine so the matter may not be linked to the mic.

Any clue ?

**Comment by Scraggarax** [comment, marked as answer]
This seems like something similar #653. And it's been reported a few times lately.

What I think might be happening, is that when the services are stopped for an update or a sudden loss of power, the analysis service just hangs or needs a restart.

I'm not entirely sure on this part, but sometimes it locks up if I'm not viewing the log as it does it at the same time.


Go into your services, and restart the analysis services. Then go to the log and watch the date and time on the right. You'll see where it's at and just keep an eye on that log until it caches up. If it's been offline for a few days, you're going to have to restart the analysis service a few times to get this working again.


Be sure to do this before your birdnet fills up with recordings. Once it does that, it will not have enough free space to do anything else. You're going to have to go into the file manager and delete the audio files from the birdsong folder to make enough space to do anything.

**Reply by Fish31520** [reply]
@Scraggarax Thanks a lot, you brought me the answer, I restarted few core services and voila, everything is back to normal !

Thanks !
